docs/source/conf.py

```diff
@@ -18,11 +18,11 @@
 # -- Project information -----------------------------------------------------
 
 project = 'tangelo'
-copyright = '2022 Good Chemistry Company'
+copyright = '2023 Good Chemistry Company'
 author = 'Valentin senicourt, Alexandre Fleury, Ryan Day, James Brown'
 
 # The full version, including alpha/beta/rc tags
-release = '0.3.1'
+release = '0.3.4'
```

Comment:
Oh: think you can grab the  version string `_version` from inside the `tangelo` package ?
No worries if not, this is really a detail.

Reply:
I can try something!